stable/jenkins/templates/jenkins-master-ingress.yaml

```diff
@@ -13,8 +13,12 @@ metadata:
   name: {{ template "jenkins.fullname" . }}
 spec:
   rules:
+{{- if .Values.Master.HostName }}
   - host: {{ .Values.Master.HostName | quote }}
     http:
```

Comment:
maybe we can move the `hostname` field to the end of the section to avoid duplicating `http`?

Reply:
Ha, makes a lot more sense than what I did :)